### PR TITLE
wireless: q14 WPS feature SW-3290

### PR DESCRIPTION
**Pull request by nandishminim**
enabled wps feature on Q14

### CODE REVIEW

**Review by bmastbergen — APPROVED** [review]

File: feeds/wifi-ax/hostapd/files/wps-hotplug.sh [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+			json_load "$(ubus -S call $ubusobj wps_status)"
+			json_get_vars pbc_status
+			[ "$pbc_status" = "Timed-out" -o "$pbc_status" = "Disabled" ] && {
+				finished=1
+			}
+		done
+		sleep 1
+	done
+	if [ "$STATE" = "led_actions_off" ] ; then
+                /sbin/leds.sh led_off
```
Comment (bmastbergen) [inline feeds/wifi-ax/hostapd/files/wps-hotplug.sh]: this should still be calling led_wlan_off like the r14 version does

File: feeds/wifi-ax/hostapd/files/wps-hotplug.sh [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+        else
+                /sbin/leds.sh $STATE
```
Comment (bmastbergen) [inline feeds/wifi-ax/hostapd/files/wps-hotplug.sh]: this should still be calling led_wlan like the r14 version does

**Review by bmastbergen — CHANGES_REQUESTED** [review]
this script should be idential to the r14 version.  Then in we customize leds.sh to do the right thing for Q14

File: feeds/wifi-ax/hostapd/files/wps-hotplug.sh (hunk 2 above)
Comment (nandishminim) [reply feeds/wifi-ax/hostapd/files/wps-hotplug.sh]: modified the script as per R14

**Pull request merged**